bar-trender-front-end/src/views/FilterResults.js

```diff
@@ -17,19 +17,35 @@ function ListPage() {
   const ListLoading = withListLoading(List);
   const [appState, setAppState] = useState({
     loading: false,
-    repos: null,
+    establishments: {},
   });
+
+  //  this.state = {establisments:{}}
+
+  const filter = {
+    "filters":{
+    }
+  };
+  
   useEffect(() => {
     setAppState({ loading: true });
 
-    const apiUrl = "https://api.github.com/users/hacktivist123/repos";
+    const apiUrl = "https://develop-backend-sprint-01.herokuapp.com/v1/establishments/get";
 
-    fetch(apiUrl)
-      .then((res) => res.json())
-      .then((repos) => {
-        setAppState({ loading: false, repos: repos });
+    fetch(apiUrl, {
+      method: 'POST',
+      body: JSON.stringify(filter),
+      headers: {
+        'Content-Type': 'application/json'
+       }
+    })
+      .then(response => response.json())
+      .then(establishments => {
+        setAppState({ loading: false, establishments: establishments });
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7138516)
```suggestion
        setAppState({ loading: false, establishments });
```